Implement P256 verification via RIP-7212 precompile with Solidity fallback (#4881)
OpenZeppelin/openzeppelin-contracts: .changeset/odd-lobsters-wash.md

---
'openzeppelin-solidity': minor
---

> `P256`: Add a library for verification/recovery of Secp256r1 (Aka P256) signatures.

Reviewer: ```suggestion
`P256`: Library for verification and public key recovery of P256 (aka secp256r1) signatures.
```
